### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Jarneid**
I have updated the Parking  solution page with new header "Parking/ "Pay-as-you-go" and inserted a new paragraph "Relevant comments"

### CODE REVIEW

File: parking/README.md [hunk 1]
```diff
@@ -46,3 +46,10 @@ The same solution can of course be used to charge weekly, monthly or yearly.
    [Recurring agreements with variable amount](https://vippsas.github.io/vipps-developer-docs/docs/APIs/recurring-api/vipps-recurring-api#recurring-agreements-with-variable-amount).
    See:
    [Create a charge](https://vippsas.github.io/vipps-developer-docs/docs/APIs/recurring-api/vipps-recurring-api#create-a-charge).
+   
+   ##Relevant comments
```
Comment (cloveras) [inline parking/README.md:50]:
```suggestion
   ## Relevant comments  
```

File: parking/README.md [hunk 2]
```diff
@@ -46,3 +46,10 @@ The same solution can of course be used to charge weekly, monthly or yearly.
    [Recurring agreements with variable amount](https://vippsas.github.io/vipps-developer-docs/docs/APIs/recurring-api/vipps-recurring-api#recurring-agreements-with-variable-amount).
    See:
    [Create a charge](https://vippsas.github.io/vipps-developer-docs/docs/APIs/recurring-api/vipps-recurring-api#create-a-charge).
+   
+   ##Relevant comments  
+   * For parking and "pay-as-you-go" cases we usually recommend that you set up a Recurring agreement with variable amount and daily interval.
+   * You are able to create as many charges you want within the interval, but we recommend that you "sum up" the usage over the day and create one charge      for the day.
```
Comment (cloveras) [inline parking/README.md:52]:
```suggestion
   * You are able to create as many charges you want within the interval, but we recommend that you "sum up" the usage over the day and create one charge for the day.
```

File: parking/README.md [hunk 3]
```diff
@@ -46,3 +46,10 @@ The same solution can of course be used to charge weekly, monthly or yearly.
...
+   * You need to take the "maximum amount" limit into account. For example if the agreement is set to "daily" and maximum amount is "1000". You will not be    able to create charges that sum up to more than 1000 in one day. Remember that it is you as the merchant that will set a "suggested maximum amount" so      you can guide the users to a suitable limit.
```
Comment (cloveras) [inline parking/README.md:53]:
```suggestion
   * You need to take the "maximum amount" limit into account. For example if the agreement is set to "daily" and maximum amount is "1000". You will not be  able to create charges that sum up to more than 1000 in one day. Remember that it is you as the merchant that will set a "suggested maximum amount" so  you can guide the users to a suitable limit. 
```

File: parking/README.md [hunk 4]
```diff
@@ -46,3 +46,10 @@ The same solution can of course be used to charge weekly, monthly or yearly.
...
    See:
    [Create a charge](https://vippsas.github.io/vipps-developer-docs/docs/APIs/recurring-api/vipps-recurring-api#create-a-charge).
+   
+   ##Relevant comments  
+   * For parking and "pay-as-you-go" cases we usually recommend that you set up a Recurring agreement with variable amount and daily interval.
+   * You are able to create as many charges you want within the interval, but we recommend that you "sum up" the usage over the day and create one charge      for the day.
+   * You need to take the "maximum amount" limit into account. For example if the agreement is set to "daily" and maximum amount is "1000". You will not be    able to create charges that sum up to more than 1000 in one day. Remember that it is you as the merchant that will set a "suggested maximum amount" so      you can guide the users to a suitable limit. 
+   * If the usage of your service sums up to more than "maximum amount" and you create a charge that is larger than "maximun amount" the end user will be      notified in Vipps and asked to increase their limit for this agreement.
```
Comment (cloveras) [inline parking/README.md:54]:
```suggestion
   * If the usage of your service sums up to more than "maximum amount" and you create a charge that is larger than "maximum amount" the end user will be notified in Vipps and asked to increase their limit for this agreement.
```

**Comment by rebekaburnett** [comment]
Let me see how it looks in Docusaurus before we merge. :-) If the sidebar gets too long, it will be a bit ugly.

**Review by rebekaburnett — APPROVED** [review]
Looks good in docusaurus. I do want to change the symbol '&' to 'and' and make a few mods to the text.

**Pull request merged**